packages/pcd/zk-eddsa-event-ticket-pcd/test/ZKEdDSAEventTicketPCD.spec.ts

```diff
@@ -222,7 +222,7 @@ describe("ZKEdDSAEventTicketPCD should work", function () {
       ticket: {
         value: serializedTicketPCD,
         argumentType: ArgumentTypeName.PCD,
-        pcdType: ZKEdDSAEventTicketPCDTypeName
+        pcdType: EdDSATicketPCDTypeName
```

Comment:
😮

Reply:
I didn't investigate deeply, but I suspect this type tag is superfluous for cases where there's no serialization involved.  Possibly also when there is, since the serialized PCD has its type in it.